client/my-sites/stats/stats-list/style.scss

```diff
@@ -558,6 +559,10 @@ ul.module-content-list-item-action-submenu {
 			margin-right: 0;
 		}
 	}
+	
+	.button.module-content-list-item-action-wrapper .gridicon {
+		vertical-align: unset; // 2
```

Comment:
I think it'd be more readable if the comment was near the style rule rather than referring to it by number:

```
// When a button instead of a link is used
.button.module-content-list-item-action-wrapper .gridicon {
		vertical-align: unset;
}
```

Reply:
👍